app/script/localization/Localizer.js

```diff
@@ -48,17 +48,52 @@ class Localizer {
 z.localization.Localizer = new Localizer();
 
 z.l10n = (() => {
-  const replaceWithString = (string, substitute) => string.replace(/{{\w+}}/, substitute);
+  const replaceTags = (string, tagSubstitutes) => {
+    Object.values(tagSubstitutes).forEach(([identifier, substitute]) => {
+      string = string.replace(new RegExp(`\\[${identifier}\\]`, 'g'), substitute);
+    });
 
-  const replaceWithObject = (string, substitutions) => {
-    Object.entries(substitutions).forEach(([identifier, substitute]) => {
+    return string;
+  };
+
+  const replaceWithArray = (string, substitutions) => {
+    substitutions.forEach(([identifier, substitute]) => {
       string = string.replace(new RegExp(`{{${identifier}}}`, 'g'), substitute);
     });
 
     return string;
   };
 
+  const replaceWithObject = (string, substitutions) => replaceWithArray(Object.entries(substitutions));
+
+  const replaceWithString = (string, substitute) => string.replace(/{{\w+}}/, substitute);
+
   return {
+    safeHtml(value, unsafeSubstitute, safeSubstitutes = {}) {
```

Comment:
The substitutions should be a single object that has the keys `safe` and `unsafe`. Cause all those scenarii are valid:
- safeHTML without any substitution;
- safeHTML with only unsafeSubstitution;
- safeHTML with only safeSubstitution;
- safeHTML with both safe and unsafe substitutions.